Calculator.html

```diff
@@ -0,0 +1,116 @@
+                                                    <button class="btm">7</button>
+                                                    <button class="btm">8</button>
+                                                    <button class="btm">9</button>
+                                                    <button class="btm equal-button">&#247;</button>
+                                                </span>
+
+                                              
+                                                        
+                                                <span class="rrow">
+                                                    <button class="btm">4</button>
+                                                    <button class="btm">5</button>
+                                                    <button class="btm">6</button>
+                                                    <button class="btm equal-button">&times; </button>
+                                                </span>
+
+                                              
+                                                    
+                                                <span class="frow">
+                                                    <button class="btm">1</button>
+                                                    <button class="btm">2</button>
+                                                    <button class="btm">3</button>
+                                                    <button class="btm equal-button">-</button>
+                                                </span>
+        
+                                               <!--Man try to do git on my own-->
+
+                                                <span class="grow">
+                                                    <button class="btm">0</button>
+                                                    <button class="btm ">.</button>
+                                                    <button class="btm equal-button" id="purp">=</button>
+                                                    <button class="btm equal-button">+</button>
+                                                               
+                                                            </span>
+
+                                                            
+                                            
+                                         
+                                        
+                                        
+
+                                        
+                                       
+
+
+
+                                        
+                                        
+                       
+
+           
+
+            
+
+
+
+
+            
+        </form>
+
+                
+        </div>                                
+                                      
+    </div>  
+</div> 
+   
+
+
+
+
+
+    </body>
+
+
+
+
+</html>
```

Comment:
Minor style issues. Make sure you format/indent your documents correctly. (-2)

---------------------
95/100 Great work!